docs/cluster-configuration.md

```diff
@@ -215,30 +215,6 @@ See [below](#configure-application-aware-quota-aaq) for Application Aware Quota
 
 **Default**: `false`
 
-### primaryUserDefinedNetworkBinding Feature Gate
-Set the `primaryUserDefinedNetworkBinding` feature gate to deploy the needed configurations for kubevirt users, so they
-can bind their VM to a user-defined network (UDN) on the VM's primary interface.
-
-**Note**: this feature is in Developer Preview.
-
-**Default**: `false`
-
-### Feature Gates Example
-
-```yaml
-apiVersion: hco.kubevirt.io/v1beta1
-kind: HyperConverged
-metadata:
-  name: kubevirt-hyperconverged
-spec:
-  infra: {}
-  workloads: {}
-  featureGates:
-    enableCommonBootImageImport: true
-    deployKubeSecondaryDNS: true
-    enableApplicationAwareQuota: true
-```
```

Comment:
This section is the example for the whole feature gates section. It shouldn't be removed. Please restore it.

Reply:
done